### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- Weather 
- Focus
- Time

image

### CODE REVIEW

**Review by dusariudaykumar — APPROVED** [review]
- code looks clean
- design looks great
- the file structure is good

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,12 @@
...
-    transform: rotate(360deg);
-  }
-}
+.background {
+  background: no-repeat center cover;
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  z-index: -1;
+  /* object-fit: contain; */
```
Comment (tushargoyal812) [inline src/App.css:11]: remove commented code

File: src/pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <>
+      <div className="main-home-container">
+        <Weather />
+      </div>
+      <div className="landing-page-container">
+        <Timer />
+        
+        <Focus />
+        {/* <button className="landing-page-btn" onClick={setNameHandler}>Continue</button> */}
```
Comment (tushargoyal812) [inline src/pages/Homepage/Homepage.jsx:17]: also here

File: src/pages/Landingpage/Landingpage.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import "./Landingpage.css"
+
+const Landingpage = () => {
+    const [username,setUsername] = useState("")
+    const userNameHandler = (e) => {
+        console.log(e.target.value);
```
Comment (tushargoyal812) [inline src/pages/Landingpage/Landingpage.jsx:7]: remove console

**Pull request merged**